ouroboros-consensus/test-storage/Test/Ouroboros/Storage/VolatileDB/StateMachine.hs

```diff
@@ -247,10 +251,19 @@ runPure dbm (CmdErr cmd err) =
         go = case cmd of
             GetBlock bid       -> do
                 Blob <$> getBlockModel tnc bid
+            GetHeader bid ->
+                Blob <$> getHeaderModel tnc bid
             GetBlockIds        ->
                 Blocks <$> getBlockIdsModel tnc
-            PutBlock b pb      ->
-                Unit <$> putBlockModel tnc err (BlockInfo b (guessSlot b) pb) (BL.lazyByteString $ toBinary (b, pb))
+            PutBlock b pb h    -> do
+                let blockInfo = BlockInfo {
+                      bbid          = b
+                    , bslot         = (guessSlot b)
+                    , bpreBid       = pb
+                    , bheaderOffset = fromIntegral headerOffset
+                    , bheaderSize   = fromIntegral headerSize
+                    }
+                Unit <$> putBlockModel tnc err blockInfo (BL.lazyByteString $ toBinary (b, pb, h))
```

Comment:
The line is still to long, make a `let`-binding for `BL.lazyByteString $ toBinary (b, pb, h)` (`blob`?).